scripts/docker/ros2/ros2_rolling_base.Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM ubuntu:focal
+
+ENV ROSDISTRO=rolling
+ENV ROS_PYTHON_VERSION=3
+ENV ROS_VERSION=2
+
+SHELL ["/bin/bash", "-c"]
+ENV DEBIAN_FRONTEND=noninteractive
+
+# setup timezone
+RUN echo 'Etc/UTC' > /etc/timezone \
+    ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime \
```

Comment:
Is there a missing `&&` before `ln -s`?

Reply:
nope, that actually copied from the official ros image and they do it like that, I thought it was weird too